src/Spago/Packages.hs

```diff
@@ -85,27 +85,48 @@ getProjectDeps :: Spago m => Config -> m [(PackageName, Package)]
 getProjectDeps Config{..} = getTransitiveDeps packageSet dependencies
 
 
+fst' :: (a, b, c) -> a
+fst' (x, _, _) = x
+
+snd' :: (a, b, c) -> b
+snd' (_, x, _) = x
+
+trd' :: (a, b, c) -> c
+trd' (_, _, x) = x
+
```

Comment:
It'd be nice if these three functions could be avoided, or at least be moved to somewhere more appropriate.